dispatcher/dispatcher.py

```diff
@@ -27,8 +36,10 @@ def setup():
     daq_config = json.loads(config['MasterDAQConfig'])
     control_mc = daqnt.get_client('daq')
     runs_mc = daqnt.get_client('runs')
-    logger = daqnt.get_daq_logger(config['LogName'], level=args.log, mc=control_mc)
+    logger = daqnt.get_daq_logger(config['LogName'], level=args.log, mc=control_mc, log_on_rotate=f'Commit {commit}')
```

Comment:
unless we change the PR, it would be `opening_message`

Reply:
Have to pad my commit count somehow